test/end-to-end/journal/journal.page.js

```diff
@@ -55,15 +58,41 @@ function JournalCorePage() {
     return $('[data-method="trial"]').click();
   }
 
+  function resetSearchFilters() {
+    const filters = new Filters();
+
+    // reset custom filters
+    filters.resetFilters();
+
+    // reset default filter negative value
+    showFullTransactions(false);
+  }
+
+  // @TODO(sfount) A significantly better selector should be used here but
+  //               the model on the component is `$ctrl.value`. Something
+  //               less ambiguos should be used.
```

Comment:
Yeah, that's a good point.  Although, since this is just a DOM selector, we could probably stick an `id` onto the bh-yes-no-radios and then subselect the containing HTML with it.  Like this:
```html
<bh-yes-no-radios
  id="showFullTransactionsToggle"
 ...>
```
```js
const radios = $('#showFullTransactionsToggle');
radios.$(`[data-choice="${selection}"]`).click();
```